js/main.js

```diff
@@ -1 +1,24 @@
-// Your functions or code that should load on all pages goes here.
+function onVersionChange(form, animation) {
+    if (typeof version_prefixes == 'undefined' || typeof fields_per_version == 'undefined'
+        || !version_prefixes || !fields_per_version) {
+        return;
+    }
+    $.each(version_prefixes, function(version, prefix) {
+        let checkbox = form.find('#id_c_versions [value="' + version + '"]');
+        $.each(fields_per_version, function(i, field_name) {
+            let input = form.find('#id_' + prefix + field_name);
+            let field = input.closest('.form-group');
+            if (checkbox.prop('checked')) {
+                field.show(animation);
+            } else {
+                field.hide(animation);
+                input.val('');
+            }
+        });
+    });
+}
+function loadVersions() {
+    let form = $('[data-form-name$="_event"]');
```

Comment:
The syntax `[foo=bar]` allows you to select HTML elements using their attributes. For example, if you have an element `<a href="#test"></a>`, you can use this selector: `[href="#test"]` or `a[href="#test"]`.

You can also use `[foo$=bar]` to say "select elements with attribute "foo" that have a value that ends with "bar"", such as `<a href="#" foo="something_bar">`.

In this case, we want to match both add event (`<form data-form-name-"add_event">...</form>`) and edit event (`<form data-form-name-"edit_event">...</form>`) forms, so we can match both forms which this selector.

FYI for your curiosity only, for "starts with" the syntax would be `[foo^=bar]` and "contains" would be `[foo*=bar]`. CSS/jQuery selectors are similar. You can read about CSS selectors [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors). jQuery selectors just have more options on top of that, listed [here](http://api.jquery.com/category/selectors/).